core/v2/types.go

```diff
@@ -321,7 +321,7 @@ type Attestation struct {
 	Sigma *core.Signature
 	// QuorumNumbers contains the quorums relevant for the attestation
 	QuorumNumbers []core.QuorumID
-	// QuorumResults contains the results of the quorum verification
+	// QuorumResults contains the operators' total signing percentage of the quorum
```

Comment:
```suggestion
	// QuorumResults is a map from quorumID to the operators' total signing percentage for that quorum.
```

Reply:
which quorums have an entry? All the quorums found in the blobHeaders for the given batch?

Reply:
each quorum has its own signing rate and different quorums may have different rates